docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -89,12 +89,21 @@ At the same time, using a NuGet package means taking a dependency on it, so you
 
     ![Download statistics on a package's listing page](media/Finding-03-Downloads.png)
 
-- *GitHub Usage*: on the package page, the **GitHub Usage** section lists public GitHub repositories that depend on this package and that have a high number of stars on GitHub. A GitHub repository's number of stars generally indicates how popular that repository is with GitHub users (more stars usually means more popular). Please visit [GitHub's Getting Started page](https://help.github.com/en/github/getting-started-with-github/saving-repositories-with-stars#about-stars) for more information on GitHub's star and repository ranking system.
+- *Used By*: on the package page, the **Used By** section lists the top 5 most popular NuGet.org packages and popular GitHub repositories that depend on this package. Dependent packages and repos can be seen as "endorsements" of this package, as package authors have chosen to trust and depend on it.
```

Comment:
It seems weird to me to have the information in an unordered list. For documentation, isn't multiple sizes (levels) of headings a more standard and readable way of structuring information like this?

Reply:
Perhaps define or compare "dependent" as a "reverse dependency" or "a package that is depended on by another, the opposite relationship of a package dependency".

I don't know how clear this would be for someone that has English, say, as a second language.

Reply:
I don't know that I want to restructure the page right now, but I'll make a note of it for the future. But, I did bold the item labels as opposed to italics which is a little more visually noticeable and consistent with the rest of the page. 

I also added a definition for "dependent" to be extra clear  😊